lib/mpl_toolkits/axes_grid1/parasite_axes.py

```diff
@@ -117,16 +165,29 @@ def parasite_axes_auxtrans_class_factory(axes_class=None):
                 {'name': 'parasite_axes'})
 
 
-ParasiteAxesAuxTrans = parasite_axes_auxtrans_class_factory(ParasiteAxes)
+# Also deprecated.
+with cbook._suppress_matplotlib_deprecation_warning():
+    ParasiteAxesAuxTrans = parasite_axes_auxtrans_class_factory(ParasiteAxes)
 
 
 class HostAxesBase:
     def __init__(self, *args, **kwargs):
         self.parasites = []
         super().__init__(*args, **kwargs)
 
-    def get_aux_axes(self, tr, viewlim_mode="equal", axes_class=ParasiteAxes):
-        parasite_axes_class = parasite_axes_auxtrans_class_factory(axes_class)
+    def get_aux_axes(self, tr=None, viewlim_mode="equal", axes_class=Axes):
+        """
+        Add a parasite axes to this host.
+
+        Despite this method's name, this should actually be thought of as an
```

Comment:
Should probably add `add_parasite_axes()` in a followup PR and (soft-)deprecate `get_aux_axes`.

Reply:
Let's do this in another PR, as you say.